documents-webapp/src/main/webapp/vue-app/documents/components/body/views/DocumentsFolderView.vue

```diff
@@ -172,7 +172,7 @@ export default {
     this.$root.$on('documents-filter', this.updateFilter);
   },
   mounted(){
-    this.$documentsUtils.injectSortTooltip(this.$t('documents.sort.tooltip'),'tooltip-marker');
+    this.$documentsUtils?.injectSortTooltip(this.$t('documents.sort.tooltip'),'tooltip-marker');
```

Comment:
Could you remove this as it was fixed by https://github.com/exoplatform/documents/commit/a65c3f0d68f884ddb1a1c4303132abde01939a4a

Reply:
done!